### PR TITLE
Added location support to elasticsearch driver using using geopoint

### PR DESCRIPTION
**Pull request by dmyers**
Only thing I'm having trouble with is chaining `search()`, `where()`, and `whereLocation()` conditions at the same time.

In order to get this to work you have to create an index like so:

```php
$fields = [
	'name' => $name,
	'_geoloc' => [
		'lat' => $lat,
		'lon' => $long,
	],
];

\Search::insert($id, $fields);
```

### CODE REVIEW

File: src/Mmanos/Search/Index.php [hunk 1]
```diff
@@ -124,6 +124,13 @@ public function select($columns = array('*'))
 	}
 	
 	/**
+	 * Create the index.
+	 *
+	 * @return bool
+	 */
+	public function createIndex() {}
```
Comment (mmanos) [inline src/Mmanos/Search/Index.php]: Why is this needed?

Comment (dmyers) [inline src/Mmanos/Search/Index.php]: I thought maybe for the driver pattern that all the drivers should have it?
